$OpenBSD: patch-src_settings_h,v 1.1 2012/08/08 12:32:10 dcoppa Exp $

Fix columns generation without configuration file (the text of the
playlist disappeared in columns display mode when there is no config
file)
(upstream git commit 6126def6c5b339c7f43d99c47501dde21d4943b3)

--- src/settings.h.orig	Sun Apr  1 18:49:09 2012
+++ src/settings.h	Wed Aug  8 13:41:05 2012
@@ -149,6 +149,7 @@ struct NcmpcppConfig
 	
 	void SetDefaults();
 	void Read();
+	void GenerateColumns();
 	
 	std::string ncmpcpp_directory;
 	std::string lyrics_directory;
